### PR TITLE
upgrade logback version

### PR DESCRIPTION
**Pull request by dgzpg**
# Description

upgrade logback version

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by dgzpg** [comment]
@kevinten10   upgrade logback version

**Comment by kevinten10** [comment]
LGTM. I will merge this in the next release.

**Comment by dgzpg** [comment]
> LGTM. I will merge this in the next release.

OK , Cool~

**Comment by dgzpg** [comment]
@kevinten10

File: sdk-spi/src/main/java/group/rxcloud/capa/spi/log/manager/LogManager.java [hunk 1]
```diff
@@ -78,35 +78,35 @@ public static Boolean whetherLogsCanOutput(CapaLogLevel capaLogLevel) {
         }
         switch (capaLogLevel) {
             case ALL:
-                return LogConfiguration.containsKey(ALL_LEVEL_SWITCH_NAME)
+                return !LogConfiguration.containsKey(ALL_LEVEL_SWITCH_NAME)
```
Comment (kevinten10) [inline sdk-spi/src/main/java/group/rxcloud/capa/spi/log/manager/LogManager.java:81]: 逆向思维 o(￣▽￣)ｄ

**Review by kevinten10 — COMMENTED** [review]
review

**Pull request merged**